src/daemon/rpmostreed-daemon.c

```diff
@@ -22,15 +22,18 @@
 #include "rpmostreed-sysroot.h"
 #include "rpmostreed-types.h"
 #include "rpmostreed-utils.h"
+#include "rpmostree-util.h"
 
 #include <libglnx.h>
 #include <systemd/sd-journal.h>
 #include <systemd/sd-daemon.h>
 #include <stdio.h>
 
 #define RPMOSTREE_MESSAGE_TRANSACTION_STARTED SD_ID128_MAKE(d5,be,a3,7a,8f,c8,4f,f5,9d,bc,fd,79,17,7b,7d,f8)
-/* I picked this arbitrarily */
-#define IDLE_EXIT_TIMEOUT_SECONDS 10
+#define IDLE_EXIT_TIMEOUT_SECONDS 60
+
+#define RPMOSTREED_CONF SYSCONFDIR "/rpm-ostreed.conf"
```

Comment:
Is there something we could put in the config file so that having a config file could land before automatic updates?

Reply:
Yeah, was thinking about that. One thought was the idle exit timeout, though not sure how useful that is for users to be able to change. Or maybe just a boolean to enable/disable idle exit? That'd be cleaner to use than `RPMOSTREE_DEBUG_DISABLE_DAEMON_IDLE_EXIT`.